.github/workflows/build_packages.yaml

```diff
@@ -240,18 +229,9 @@ jobs:
           # - opensuse
           - centos8
           - centos7
-          - centos6
```

Comment:
Do we not build centos6 packages anymore? Maybe we should talk to our business colleagues about this move
By the way, I think we can stop building centos8 packages, Red Hat announced that centos8 will stop being supported on December 31

Reply:
for 5.0, yes. the decision has been made based on the data collected from telemetry
there are only 2 users on centos6.

Reply:
centos 7 and 8, we'll keep building for now.